### ISSUE TITLE
Auto-placement

### ISSUE THREAD

**Issue by JeanCollas** [Sphinxxxx/vanilla-picker#8]
Handling the placement would be great (probably depending if the click is further than half of the screen size). Right now the picker can get hidden off the screen

![image](https://user-images.githubusercontent.com/3519393/49913586-58987e00-fe8e-11e8-9fa5-ff3b6af98266.png)

**Comment by Sphinxxxx** [comment]
Yes, that would be a nice feature! I'll just leave this open for now.
